src/main/java/com/cisco/trex/stateful/model/stats/LatencyData.java

```diff
@@ -0,0 +1,51 @@
+package com.cisco.trex.stateful.model.stats;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Map;
+
+public class LatencyData {
+
+    @JsonProperty("cpu_util")
+    public String cpu_util;
+
+    @JsonProperty("epoch")
+    public Integer epoch;
+
+    public Map<Integer, PortLatencyData> portLatencyDataMap;
+
+    @JsonProperty("unknown")
+    public Integer unknown;
+
```

Comment:
it's better to add @JsonProperty(key) annotation for all set/getXXX if it has corresponding field in json response.

Reply:
ok, I will adjust it.